instagram/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
It's a good habit to delete these files if you're not making your app progressive, this isn't really doing anything.